{
  "specName": "cypress%2Fe2e%2Fopptjeningforside_hele.cy.js",
  "title": "forside",
  "suiteName": "Opptjening-forside",
  "testName": "Opptjening-forside forside",
  "testError": "The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > \n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.",
  "testCommands": [
    "viewport 1000, 660",
    "visit /pensjon/opptjening",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "request /pensjon/opptjening, {log: true}",
    "get #forklartseksjon",
    "click ",
    "uncaught exception Error: "
  ]
}
